fdbserver/ClusterController.actor.cpp

```diff
@@ -147,6 +149,28 @@ struct BlobManagerSingleton : Singleton<BlobManagerInterface> {
 	void recruit(ClusterControllerData* cc) const { cc->recruitBlobManager.set(true); }
 };
 
+struct EncryptKeyProxySingleton : Singleton<EncryptKeyProxyInterface> {
+
+	EncryptKeyProxySingleton(const Optional<EncryptKeyProxyInterface>& interface) : Singleton(interface) {}
+
+	Role getRole() const { return Role::ENCRYPT_KEY_PROXY; }
+	ProcessClass::ClusterRole getClusterRole() const { return ProcessClass::EncryptKeyProxy; }
+
+	void setInterfaceToDbInfo(ClusterControllerData* cc) const {
+		if (interface.present()) {
+			TraceEvent("CCEKP_SetInf", cc->id).detail("Id", interface.get().id());
```

Comment:
the other singletons do not have logging in the same location, maybe either add tracing to all of them or remove to make consistent

Reply:
I feel information added is useful for debugging, hence, enhanced other singleton processes as well.
Thanks for pointing it out.